### PR TITLE
(fix): Update installation steps in README.md

### PR DESCRIPTION
**Pull request by kmjayadeep**
changes:

- Added setup instructions using docker and docker compose
- Changed nodejs installation instruction to download version 12.x instead of 9.x

Signed-off-by: Jayadeep KM <kmjayadeep@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
-### Install Yarn
+### Run the server
 
 ```bash
-npm install -g yarn
+docker-compose up
 ```
 
-### Clone litmus-docs repository
+## Manual Setup
```
Comment (inyee786) [inline README.md]: can put this step at above, people usually use this for local development and docker-compose is a way of testing the thing, not for development

**Comment by inyee786** [comment]
@qiell do we need `package-lock.json` file, as for reactjs project many people use yarnpkg instead of npm

**Comment by kmjayadeep** [comment]
> @qiell do we need `package-lock.json` file, as for reactjs project many people use yarnpkg instead of npm

These days `npm` are also being used extensively as it supports many of the features `yarn` has. In any case we should use either `npm` or `yarn` but not both. I will create separate PR to make similar change in dockerfile as well.

File: README.md (hunk 1 above)
Comment (kmjayadeep) [reply README.md]: done

**Review by ksatchit — DISMISSED** [review]
/lgtm

**Review by chandankumar4 — APPROVED** [review]

**Review by chandankumar4 — CHANGES_REQUESTED** [review]
@kmjayadeep, I think we don't require these changes, Can you take a look at this. https://github.com/openebs/openebs-docs (This is also one of the docusaurus projects)

**Comment by kmjayadeep** [comment]
> @kmjayadeep, I think we don't require these changes, Can you take a look at this. https://github.com/openebs/openebs-docs (This is also one of the docusaurus projects)

Hi @chandankumar4 

1. In openebs-docs also, there are similar issues. `npm install -g yarn` command is provided in readme, but the project is using `npm` and not `yarn`. This can cause conflicts.
2. `package-lock.json` is missing. This is important to be able to resolve to correct dependencies while doing `npm install`
3. `docker-compose` instructions are optional, but it makes development easier

**Review by chandankumar4 — DISMISSED** [review]
LGTM
Thanks, @kmjayadeep for your contribution, I updated some of the changes as per required!

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**